### PR TITLE
[New] Added test cases for pages admin form builder components transl…

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for pages admin form builder components translations form components field translation component.

- component.spec.js

### CODE REVIEW

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js [hunk 1]
```diff
@@ -28,26 +26,16 @@ describe("<FieldTranslationRow />", () => {
...
-      "fields.field_1.display_name.en",
-      "fields.field_1.display_name.fr",
-      "fields.field_1.display_name.ar"
-    ];
-
-    const fields = component.find(FormSectionField).map(f => f.props().field.get("name"));
 
-    expect(fields).to.deep.equal(expectedNames);
+    mountedFormComponent(<FieldTranslationRow {...props} />, state)
+    expect(screen.getAllByRole('textbox')).toHaveLength(1);
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js:39]: At least 2 inputs should exist in a FieldTranslationRow one for the English language and one for the selected language.

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js [hunk 2]
```diff
@@ -28,26 +29,17 @@ describe("<FieldTranslationRow />", () => {
...
+  const props = { field: fromJS(field1), selectedLocaleId: "en" };
 
   it("should render <FieldTranslationRow />", () => {
-    expect(component.find(FieldTranslationRow)).to.have.lengthOf(1);
+    localesToRender.mockReturnValue([fromJS({ id: "en" })]);
+    mountedFormComponent(<FieldTranslationRow {...props} />, state);
+    expect(screen.getByText("forms.manage")).toBeInTheDocument();
   });
 
   it("should render the <FormSectionField /> for the available languages", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js:40]: This test should test if there is a form section field for en, ar, and fr locales. There should be 3 inputs with different name attributes
```
      "fields.field_1.display_name.en",
      "fields.field_1.display_name.fr",
      "fields.field_1.display_name.ar"

```

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js [hunk 3]
```diff
@@ -28,26 +29,17 @@ describe("<FieldTranslationRow />", () => {
...
 
-  beforeEach(() => {
-    ({ component } = setupMockFormComponent(FieldTranslationRow, {
-      props: { field: fromJS(field1), selectedLocaleId: "fr" },
-      state
-    }));
-  });
+  const props = { field: fromJS(field1), selectedLocaleId: "en" };
 
   it("should render <FieldTranslationRow />", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js:34]: Remove this unit test

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js (hunk 2 above)
Comment (jtoliver-quoin) [reply app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js:40]: Looks like the return of localesToRender is returning an plain array. You can remove the fromJS

**Review by jtoliver-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js]: Done

File: app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js (hunk 3 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js]: Done

**Comment by dhernandez-quoin** [comment]
Already merged here: https://github.com/primeroIMS/primero/pull/454

**Pull request closed without merging**